app/src/main/java/com/example/weightcontrolapp/MainActivity.kt

```diff
@@ -14,6 +15,10 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
         setSupportActionBar(findViewById(R.id.toolbar))
 
+//        supportFragmentManager.beginTransaction()
+//            .replace(R.id.WeightRecordFragment, WeightRecordFragment())
+//            .commit()
+
         findViewById<FloatingActionButton>(R.id.fab).setOnClickListener { view ->
             Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                     .setAction("Action", null).show()
```

Comment:
These codes are necessary for this app.

Reply:
[ee17e5c](https://github.com/kenta7777/WeightControlApp/pull/1/commits/ee17e5cfdc52e44e453fbb26e6cc82fd2a3552db)